binstar_client/utils/__init__.py

```diff
@@ -247,8 +245,8 @@ def recursive_update(d, u):
             d[k] = u[k]
     return d
 
-DEFAULT_URL = 'https://api.anaconda.org'
```

Comment:
Don't understand why. If we replace `DEFAUL_URL` we will be replacing everywhere.